lib/index.js

```diff
@@ -59,3 +77,31 @@ exports.register.attributes = {
   version: '1.0.0',
   multiple: true
 };
+
+internals.globCallback = function (files) {
+  for (var i = 0, il = files.length; i < il; i++) {
+    internals._addModel(files[i]);
+  }
+};
+
+internals._addModel = function (model) {
```

Comment:
Since you only use this function once and it's only one line, you can just call `internals.modelObjects.push(model);` directly.